packages/api/src/lib/granule-delete.ts

```diff
@@ -145,7 +147,7 @@ const deleteGranuleAndFiles = async (params: {
         deletedFiles: files,
       };
     } catch (error) {
```

Comment:
When granule is not in dynamoDB, delete has error:
{ "error": { "name": "TypeError", "message": "Cannot read properties of undefined (reading 'files')", "stack": [ "TypeError: Cannot read properties of undefined (reading 'files')", " at deleteGranuleAndFiles (/var/task/webpack:/granule-delete.ts:148:119)", " at processTicksAndRejections (node:internal/process/task_queues:96:5)", " at del (/var/task/webpack:/endpoints/granules.js:551:27)" ] }, "level": "error", "message": "Cannot read properties of undefined (reading 'files')", "sender": "@cumulus/api", "timestamp": "2023-04-18T20:35:10.277Z" }